src/components/MapView/Analyser/index.tsx

```diff
@@ -348,7 +400,7 @@ function Analyser({ classes }: AnalyserProps) {
             {!analysisResult && (
               <Button
                 className={classes.innerAnalysisButton}
-                onClick={runAnalyser}
+                onClick={() => runAnalyser()}
```

Comment:
is that change necessary?

Reply:
It wasn't working initially, so I had to use an anonymous function which fixed the issue but reverted it back just now and it worked fine.